plutus-core/plutus-ir/test/transform/letFloatIn/avoid-floating-into-RHS

```diff
@@ -0,0 +1,22 @@
+(let
+  (nonrec)
+  (termbind
+    (strict)
+    (vardecl addInteger (fun (con integer) (fun (con integer) (con integer))))
+    (builtin addInteger)
+  )
+  (termbind
```

Comment:
We might as well simplify these, even if they're generated. I'm still reading them! You can just delete both of these bindings and put `(builtin addInteger)` everywhere.

Reply:
Same in the other tests